presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -944,7 +944,7 @@ private static OptionalDouble getAverageColumnLength(OptionalLong totalSizeInByt
             return ImmutableSet.of(NUMBER_OF_NON_NULL_VALUES, TOTAL_SIZE_IN_BYTES, MAX_VALUE_SIZE_IN_BYTES);
         }
         if (type instanceof ArrayType || type instanceof RowType || type instanceof MapType) {
-            return ImmutableSet.of();
+            return ImmutableSet.of(NUMBER_OF_NON_NULL_VALUES, TOTAL_SIZE_IN_BYTES);
```

Comment:
Could you please extract it into a separate commit? Also could you please add the tests to the `TestHiveIntegrationSmokeTest` (cases in `testCollectColumnStatisticsOnCreateTable`, `testCollectColumnStatisticsOnInsert`, `testAnalyzePartitionedTable`, `testAnalyzeUnpartitionedTable`)